chain-core/src/init/config.rs

```diff
@@ -42,6 +44,18 @@ pub struct JailingParameters {
     pub missed_block_threshold: u16,
 }
 
+#[derive(Debug, PartialEq, Eq, Clone, Copy, Encode, Decode)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct SlashingParameters {
+    /// Percentage of funds (bonded + unbonded) slashed when validator is not live (liveness is calculated by jailing
+    /// parameters)
+    pub liveness_slash_percent: u8,
+    /// Percentage of funds (bonded + unbonded) slashed when validator makes a byzantine fault
+    pub byzantine_slash_percent: u8,
```

Comment:
As one will need to do fixed-decimal calculations with these, wouldn't make sense to have a dedicated datatype directly -- e.g. wrap `Milli`: `pub struct MaxSlashRate(Milli);` where constructor / decoder of MaxSlashRate would check the value is in the range [0.0000, 1.0000) ?